drivers/windows/R/batch.R

```diff
@@ -60,12 +90,17 @@ template_data_common <- function(config, path_root) {
     "ECHO Removing mapping {{drive}}\nnet use {{drive}} /delete /y",
     network_shares_data)
 
+  r_version <- config$r_version
+  if (config$platform != "linux") {
+    r_version <- version_string(config$r_version)
+  }
+
   list(
     hostname = hipercow:::hostname(),
     date = as.character(Sys.time()),
     hipercow_version = hipercow_version(),
     hipercow_windows_version = hipercow_windows_version(),
-    r_version = version_string(config$r_version),
+    r_version = r_version,
```

Comment:
This change seems odd, given the odd change above? Half way through refactoring?

Reply:
r_version at this point is the correct thing for either OS.